cluster/update-storage-objects.sh

```diff
@@ -59,7 +61,12 @@ for resource in "${resources[@]}"
 do
   for namespace in "${namespaces[@]}"
   do
+    # If get fails, assume it's because the resource hasn't been installed in the apiserver.
+    # TODO hopefully we can remove this once we use dynamic discovery of gettable/updateable
+    # resources.
+    set +e
```

Comment:
Getting at least one resource isn't a requirement. You could have a cluster with zero pods (or whatever) and this script should not fail. Do you want me to try to differentiate between a successful get (0 or more) vs "resource not installed" (not an error) vs "some other error" (fail)?